src/Command/RabbitUpdateAllAcceptedDatasetDOICommand.php

```diff
@@ -2,6 +2,7 @@
 
 namespace App\Command;
 
+use App\Message\DoiMessage;
```

Comment:
Also rename command to not mention Rabbit.

Reply:
Renamed it